linux_pipeline/Jenkinsfile_container_images

```diff
@@ -226,5 +227,20 @@ pipeline {
           }
         }
       }
+      stage('Publish artifacts') {
+        agent {
+          node {
+            label 'lis-f2334'
```

Comment:
change the label name to something relevant like windows_meta_slave, not the actual host names